File: Examples/CSharp/Programming-Documents/Sections/DeleteSectionContent.cs
using System;
using System.Collections.Generic;
using Aspose.Words;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Sections
{
    class DeleteSectionContent
    {
        public static void Run()
        {
            //ExStart:DeleteSectionContent
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithSections();

            Document doc = new Document(dataDir + "Document.doc");
            Section section = doc.Sections[0];
            section.ClearContent();
            //ExEnd:DeleteSectionContent
            Console.WriteLine("\nSection content at 0 index deleted successfully.");
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Sections/DeleteSectionContent.vb
Imports Microsoft.VisualBasic
Imports Aspose.Words
Public Class DeleteSectionContent
    Public Shared Sub Run()
        ' ExStart:DeleteSectionContent
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithSections()

        Dim doc As New Document(dataDir & Convert.ToString("Document.doc"))
        Dim section As Section = doc.Sections(0)
        section.ClearContent()
        ' ExEnd:DeleteSectionContent
        Console.WriteLine(vbLf & "Section content at 0 index deleted successfully.")
    End Sub
End Class
